cypress/plugins/index.js

```diff
@@ -12,7 +12,6 @@
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
 
-const { logger } = require("webpack-cli/lib/utils");
```

Comment:
is webpack logger deprecated now?

Reply:
Nope I don't think it is. It was giving issues saying it would find this specific utility. Out of curiosity what's the benefit of using this over the default cypress logger?

Reply:
oh, sorry, I didn't even realise we're in cypress.. ignore, what you've got is fine